assets/css/map-segment.scss

```diff
@@ -0,0 +1,11 @@
+.map-segment {
+  position: absolute;
+  left: 50%;
+  top: 50%;
+  transform: translateY(-50%) translateX(-50%);
+  z-index: 999;
+
+  &--blue {
+    filter: drop-shadow(0px 0px 8px rgba(46, 62, 77, 0.4));
+  }
```

Comment:
The stick color is also different for the blue line (unlike the other lines). Should be `#004BCC` while the route pill color is `#003DA5`.

Just out of curiosity, where does this drop-shadow live in the spec? I see the comment saying there should be a drop-shadow, but I can't find the values in the spec

Reply:
Yeesh, so subtly but I still shouldn't have missed that. Unfortunately, I think that means we can't reliably use that util function anymore. We can't change it because OFT uses it. I made a new function in the component itself. What do you think about that?

Also, the drop-shadow in design can be found [here](https://www.figma.com/file/AXmojXuKz4TRVhznOIMuBJ/Screenplay-Design?node-id=1149%3A54197).